File: Semaine_1/Colors_C/Colors_C/ViewController.m
//
//  ViewController.m
//  Couleur_C
//
//  Created by Robin Blottière--Mayo on 04/09/2018.
//  Copyright © 2018 Robin Blottière--Mayo. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
	[super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
	[super didReceiveMemoryWarning];
	// Dispose of any resources that can be recreated.
}

- (IBAction)ChangeColorStepper:(id)sender {
	int v = (int)_ColorOutletStepper.value;
	switch (v) {
		case 0:
			[_ColorView setBackgroundColor:(UIColor.yellowColor)];
			break;
		case 1:
			[_ColorView setBackgroundColor:(UIColor.redColor)];
			break;
		case 2:
			[_ColorView setBackgroundColor:(UIColor.greenColor)];
			break;
		case 3:
			[_ColorView setBackgroundColor:(UIColor.blueColor)];
			break;
		case 4:
			[_ColorView setBackgroundColor:(UIColor.blackColor)];
			break;
		case 5:
			[_ColorView setBackgroundColor:(UIColor.cyanColor)];
			break;
		case 6:
			[_ColorView setBackgroundColor:(UIColor.magentaColor)];
			break;
		default:
			[_ColorView setBackgroundColor:(UIColor.brownColor)];
			break;
	}
}

@end

File: Semaine_1/Colors_S/Colors_S/ViewController.swift
//
//  ViewController.swift
//  Couleur_S
//
//  Created by Robin Blottière--Mayo on 02/09/2018.
//  Copyright © 2018 Robin Blottière--Mayo. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

	override func viewDidLoad() {
		super.viewDidLoad()
		// Do any additional setup after loading the view, typically from a nib.
	}

	override func didReceiveMemoryWarning() {
		super.didReceiveMemoryWarning()
		// Dispose of any resources that can be recreated.
	}
	
	@IBOutlet weak var ColorView: UIView!
	@IBOutlet weak var ChangeColorStepperOutlet: UIStepper!
	
	@IBAction func ChangeColorStepper(_ sender: Any) {
		switch ChangeColorStepperOutlet.value {
		case 0.0:
			ColorView.backgroundColor = UIColor.yellow
		case 1.0:
			ColorView.backgroundColor = UIColor.red
		case 2.0:
			ColorView.backgroundColor = UIColor.green
		case 3.0:
			ColorView.backgroundColor = UIColor.blue
		case 4.0:
			ColorView.backgroundColor = UIColor.black
		case 5.0:
			ColorView.backgroundColor = UIColor.cyan
		case 6.0:
			ColorView.backgroundColor = UIColor.magenta
		default:
			ColorView.backgroundColor = UIColor.yellow
		}
	}
}
